src/single-spa-vue.js

```diff
@@ -44,6 +44,10 @@ export default function singleSpaVue(userOpts) {
     });
   };
 
+  if (opts.appOptions.router) {
+    opts._startRoute = opts.appOptions.router.history.current;
```

Comment:
What if `router.history.current` doesn't exists? I think there should be a warning in this case...